manifests/vcsscm/git.pp

```diff
@@ -45,6 +47,79 @@
           source => 'puppet:///modules/software/vcsscm/git/bash-git-prompt',
         }
       }
+
+      if $gitconfig or $gitignore {
+        # Ensure target path (avoiding to manage base path)
+        exec { 'mkdir -p /etc/skel/.config':
```

Comment:
Where was the file resource repeated? I only see this once. How were you getting this duplicated.
Was it in your profile or something? Or in another module?
Typically this is handled by the `ensure_resource()` in the stdlib module: https://github.com/puppetlabs/puppetlabs-stdlib#ensure_resource